pages/rops/procedures/index.js

```diff
@@ -17,6 +17,17 @@ module.exports = () => {
       .catch(next);
   });
 
+  app.post('/unsubmit', (req, res, next) => {
+    const params = {
+      method: 'POST'
+    };
+    req.api(`/establishment/${req.establishmentId}/project/${req.projectId}/rop/${req.ropId}/unsubmit`, params)
+      .then(() => {
+        res.redirect(req.buildRoute('rops.procedures'));
```

Comment:
Do we want to add a toaster message here?

Reply:
a) Yes
b) `Return updated.`

Reply:
👍